process/subprocesses/ghsci.py

```diff
@@ -1136,45 +1130,59 @@ def _create_open_space_areas(self):
 
         open_space_areas_setup(self.codename)
         return 'Open space areas created.'
+    
+    def _compute_lpugs(self):
+        """Identify large public urban green space for this study region."""
+        from _07_large_public_urban_green_space import large_public_urban_green_space
+
+        large_public_urban_green_space(self.codename)
+        return 'Large public urban green space analysis completed.'
+    
+    def _compute_guhvi(self):
+        """Generate global urban heat vulnerability index."""

```

Comment:
Because these function docstrings are displayed when someone types `r.help()`, it would be worth mentioning something like `"(optional; set 'gee = true' in configuration file and use global-indicators.ee.bat or global-indicators.ee.sh launcher to run analysis)"`